iothub/device/tests/ClientPropertyCollectionTests.cs

```diff
@@ -255,10 +315,13 @@ public void ClientPropertyCollection_CanAddNullPropertyWithComponentAndGetBackWi
         [TestMethod]
         public void ClientPropertyCollection_CanAddMultiplePropertyWithComponentAndGetBackWithoutDeviceClient()
         {
+            // arrange
             var clientProperties = new ClientPropertyCollection();
             clientProperties.AddComponentProperty(ComponentName, StringPropertyName, StringPropertyValue);
             clientProperties.AddComponentProperty(ComponentName, IntPropertyName, IntPropertyValue);
 
+            // act, assert
+
```

Comment:
nit: all of these act, assert comments have an extra line? Intentional?

Reply:
Yes, wherever we have a group of act-assert statements in a single test method, I've added the "act, assert" comment at the top and then left a newline to indicate that the comment is for the next set of statements, instead of being for the immediate following statement only.